### PR TITLE
DG draft

### PR DESCRIPTION
**Pull request by wadia82**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -277,6 +277,26 @@ Alternatives:
...
 
+
+###Delete Feature
+
+####Current Implementation
+
+The delete mechanism deletes a patient, appointment or bill identified by their index in the list. The deletion is done
+through the `deletePatient`, `deleteAppointment` and `deleteBill` functions in `ModelManager`. 
+
+Given below is an example usage scenario and how the find mechanism behaves at each step.
```
Comment (guokweijie) [inline docs/DeveloperGuide.md:288]: delete mechanism*

**Review by guokweijie — COMMENTED** [review]
just a small typo

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -277,6 +277,26 @@ Alternatives:
...
+
+Given below is an example usage scenario and how the delete mechanism behaves at each step.
+
+Step 1. The user launches the application. All patients, appointments and bills are shown on different sections
+of the application as indexed lists.
+
+Step 2. The user executes `deletePatient 2` command to delete the patient at index 2 in the list.
+The `delete` command calls `Model#deletePatient` to delete the patient from the list of patients.
+
+The delete feature is now seperated for the patients, appointments and bills sections.
```
Comment (TZL0) [inline docs/DeveloperGuide.md:296]: Is it better to mention that the DeletePatientCommand deletes the relative appointments and bills?

Comment (guokweijie) [reply docs/DeveloperGuide.md:296]: But it doesn't right? I thought it was seperate.

**Review by TZL0 — APPROVED** [review]

**Pull request merged**